src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -201,6 +201,7 @@ class PillSelector extends React.Component<Props, State> {
         const ariaAttrs = {
             'aria-invalid': hasError,
             'aria-errormessage': this.errorMessageID,
+            'aria-describedBy': this.errorMessageID,
```

Comment:
minor: lowercase `aria-describedby`

Also - what happens when there is no error? is this just empty or undefined?

Reply:
@swfree I just changed the` aria-describedby` attribute. This `errorMessageID` is an `uniqueId`, it means , this variable is always  created when the component is created